src/Modal.php

```diff
@@ -511,11 +511,29 @@ public function title(string|Stringable $content, string $tag = 'H5', array $att
      * ```php
      * $modal->triggerButton('Launch Modal');
      * ```
+     *
+     * Note: Setting `aria-hidden="true"` may cause accessibility errors in some modern browsers (for example,
+     * Chrome Beta/Canary) when the modal is closed, due to focus management issues during the fade transition.
+     * If this occurs, set `$ariaHidden` to `false` and manage the attribute via JavaScript, for example, by updating
+     * it after the `hidden.bs.modal` event.
+     *
+     * @link https://github.com/twbs/bootstrap/issues/41005 for details.
+     *
+     * ```php
+     * <script>
+     * <!-- Script to remove aria-hidden -->
+     * document.addEventListener('hide.bs.modal', (event) => {
+     *     const modal = event.target;
+     *     modal.removeAttribute('aria-hidden');
+     * });
+     * </script>
+     * ```
      */
     public function triggerButton(
         string|Stringable $content = 'Launch modal',
         bool $staticBackdrop = false,
         array $attributes = [],
+        bool $ariaHidden = true,
```

Comment:
How about not adding another parameter but checking if there's explicitly defined `aria-hidden` in `$attributes` and adding `true` only if it is not yet defined? Method signature will be simpler.